bemb/model/bayesian_coefficient.py

```diff
@@ -211,9 +254,19 @@ def log_prior(self,
 
         else:
             mu = self.prior_zero_mean
-        out = LowRankMultivariateNormal(loc=mu,
-                                        cov_factor=self.prior_cov_factor,
-                                        cov_diag=self.prior_cov_diag).log_prob(sample)
+
+        if self.distribution == 'gaussian':
+            out = LowRankMultivariateNormal(loc=mu,
+                                            cov_factor=self.prior_cov_factor,
+                                            cov_diag=self.prior_cov_diag).log_prob(sample)
+        elif self.distribution == 'gamma':
+            concentration = torch.exp(mu)
+            rate = self.prior_variance
+            out = Gamma(concentration=concentration,
+                        rate=rate).log_prob(sample)
```

Comment:
**This might be a bug!** The `rate` and `prior_variance` are different, can you double-check it?

Reply:
Yes, i meant for gamma prior_variance to represent rate. Earlier too I set prior_variance to rate. This should be correct.

Reply:
Thanks for flagging it though.

Reply:
Is `prior_variance` the **rate of prior** or the **log(rate) of prior**?